### ISSUE TITLE
Question about accuracy

### ISSUE THREAD

**Issue by straightpinexu** [meixiaoguang/SSAN#3]
Hello, I am using your code for Comparative experiment. However, when I train your model in Pavia Center with 100 training samples each class (as default setting in your code), I can only achieve the outcome below:
[oa:0.9614, aa:0.8896, kappa:0.9456] (I run your code twice)
I only change the data path in your code, and use all the default hyper-parameters in your code. 
So, could you please tell me what's going wrong?  My platform is GTX1080Ti, i7-7900,

**Comment by meixiaoguang** [comment]
i have updated yesterday, please try the latest.

**Comment by straightpinexu** [comment]
Yes, I used latest code released yesterday and get the outcome mentioned earlier. Is it necessary to send you my code?
